Mastodon.API/Link.cs

```diff
@@ -44,5 +55,29 @@ public override string ToString()
         {
             return string.Format("[Link: MaxId={0}, SinceId={1}]", MaxId, SinceId);
         }
+
+        internal static string getValueFromHeaders(string key, HttpHeaders headers)
```

Comment:
I think to be more consistent with function naming this should be GetValueFromHeaders.